spec/hydra/pcdm/models/collection_spec.rb

```diff
@@ -26,9 +26,13 @@
         subject.members << object1
         subject.members << object2
         subject.members << collection3
+        subject.save
```

Comment:
Is the call to `#save` necessary to get IDs?  I see some other specs testing for `child_object_ids` and `child_collection_ids` without needing a save.  Calling `#save` does hurt test performance a bit.